Formula/structurizr-cli.rb

```diff
@@ -0,0 +1,25 @@
+class StructurizrCli < Formula
+  desc "Command-line utility for Structurizr"
+  homepage "https://structurizr.com"
+  url "https://github.com/structurizr/cli/releases/download/v1.4.4/structurizr-cli-1.4.4.zip"
+  sha256 "ac173013c397fd2180a2b8134e29f8dfd3e0884418252433bcc9982e15b03909"
+  license "Apache-2.0"
+
+  bottle :unneeded
+
+  depends_on "openjdk"
+
+  def install
+    jar = "structurizr-cli.jar"
+    libexec.install "structurizr-cli-#{version}.jar" => jar
+    (bin/"structurizr-cli").write <<~EOS
+      #!/bin/bash
+      exec "#{Formula["openjdk"].opt_bin}/java" -jar #{libexec}/#{jar} "$@"
+    EOS
+    chmod 0755, bin/"structurizr-cli"
+  end
+
+  test do
+    shell_output bin/"structurizr-cli", 1
```

Comment:
Please use `assert_match` to check for some actual output here

Reply:
I've used `assert_equal` now, please, check if the test looks good now. I took inspiration from an existing formula.